nx.json

```diff
@@ -181,6 +178,9 @@
         "{projectRoot}/dist/middlewares/**/*.js",
         "{projectRoot}/dist/middlewares/**/*.d.ts",
         "{projectRoot}/dist/middlewares/**/*.d.ts.map",
+        "{projectRoot}/dist/ng-add/**/*.js",
```

Comment:
this `ng-add` should not be in the `dist/` folder directly but within the `schematics/` folder.
Do we have exception in the code?